src/svd/peripheral.rs

```diff
@@ -76,7 +76,7 @@ pub struct Peripheral {
 #[derive(Clone, Debug, Default, PartialEq)]
 pub struct PeripheralBuilder {
```

Comment:
Maybe it makes sense to parametrize structures with size: `PeripheralBuilder<U>` instead of replacing all u32 with u64?

Reply:
Hmm, looks like unnecessary complication, given that the size type is not something the user can change.